util/fibers/submit_entry.h

```diff
@@ -119,8 +120,8 @@ class SubmitEntry {
   void PrepFallocate(int fd, int mode, off_t offset, off_t len) {
     PrepFd(IORING_OP_FALLOCATE, fd);
     sqe_->off = offset;
-    sqe_->len = mode;
-    sqe_->addr = uint64_t(len);
+    sqe_->len = len;
```

Comment:
How did this work 🤷‍♂️

Reply:
it worked . this is the implementation of liburing library:
```
IOURINGINLINE void io_uring_prep_fallocate(struct io_uring_sqe *sqe, int fd,
					   int mode, __u64 offset, __u64 len)
{
	io_uring_prep_rw(IORING_OP_FALLOCATE, sqe, fd,
			0, (unsigned int) mode, (__u64) offset);
	sqe->addr = (__u64) len;
}

```
so I do not think yours is correct. In any case, please add coverage to util/fibers/uring_file_test.cc